src/main/java/singly_linked_list/Controller.java

```diff
@@ -0,0 +1,115 @@
+package singly_linked_list;
+import java.util.Scanner;
+
+
+public class Controller {
+    private Scanner scanner;
+    private ServiceOperations serviceOperations;
+    private Node head;
+
+    public Controller() {
+        scanner = new Scanner(System.in);
+        serviceOperations = new ListService();
+    }
+
+    public void userInterface() {
+        int choose;
+        System.out.println("1-LIFO list\n2-FIFO list\n");
+        choose = scanner.nextInt();
+        if (choose == 1) {
+            System.out.println("Enter a sequence of numbers: (input ending - number 0)");
+            head = serviceOperations.createLifo();
+        } else if (choose == 2) {
+            System.out.println("Enter a sequence of numbers: (input ending - number 0)");
+            head = serviceOperations.createFifo();
+        } else {
+            System.out.println("Incorrect number.");
+            return;
+        }
+        print();
+        while (true) {
+            System.out.println("What do you want to do:\n1-Delete item;\n2-Add element BEFORE;\n3-Add element AFTER;" +
+                    "\n4-Show all elements of a sequence;\n5-Is the list sorted in ascending order?\n" +
+                    "6-Is the list sorted in descending order?\n7-Print list recursively;\n8-Exit;");
+            choose = scanner.nextInt();
+            switch (choose) {
+                case 1 -> {
+                    print();
+                    removeElement();
+                    print();
+                }
+                case 2 -> {
+                    print();
+                    before();
+                    print();
+                }
+                case 3 -> {
+                    print();
+                    after();
+                    print();
+                }
+                case 4 -> print();
+                case 5 -> isAsc();
+                case 6 -> isDesc();
+                case 7 -> printRecursively();
+                case 8 -> System.exit(1);
+            }
+        }
+    }
+
+    private void print() {
+        serviceOperations.printList(head);
+    }
+
+    private void printRecursively(){
```

Comment:
You can use ```Ctrl+Alt+Shift+L``` to reformat code in a whole file

Reply:
Done.